docs/source/cli.rst

```diff
@@ -0,0 +1,15 @@
+Command Line Interface
+======================
+
+Dask provides a command line tool (``dask``) to accomplish a number of
+tasks.
```

Comment:
We probably want to change this to reflect the current state of this work. Maybe say that it's experimental and include the version info for when it was added?

Reply:
At this stage I don't think that we should have an experimental label in docs for something top-level and that is this important.  At least not if it's going to be in the index.  I think that we should invest in this page by adding `dask scheduler` and whatnot and then not think of it as experimental.

Reply:
Yeah I agree that is an even better idea!

Reply:
I'll third this-- I think the docs will be a large (if not the most significant) part of this PR